docs/guide/configuration.rst

```diff
@@ -46,7 +46,7 @@ that loads API files from the ``src/data`` folder of the SDK.
 credentials
 ~~~~~~~~~~~
 
-:Type: ``array|Aws\Credentials\CredentialsInterface|bool|callable``
+:Type: ``array|Aws\Credentials\CredentialsInterface|bool|callable|Aws\CacheInterface``
```

Comment:
I didn't notice they used to be in alphabetical order until I read your comment. I see no reason to break with tradition on that.